recipe/meta.yaml

```
@@ -19,8 +19,7 @@ source:
   folder: dotnet
 
 build:
-  skip: true  # [osx or win or linux]
-  number: 0
+  number: 1
```

```suggestion
  skip: true  # [osx]
  number: 1
```
I wasn't able to get OSX to work so it should be skipped (for now)

---

Yeah! I already realized, thanks for fixing it :D